### ISSUE TITLE
Dropout mode

### ISSUE THREAD

**Issue by glample** [pytorch/pytorch#477]
I'm trying to import some models from theano to pytorch, but I'm facing issues with the dropout, which has a different mode: instead of doing nothing in eval and dropping + scaling in training, in theano I had scaling in eval and dropping in training.

Torch had this option to create the dropout module in one mode or the other. Could we also have this in pytorch?

**Comment by soumith** [comment]
closing as willnotfix: https://github.com/pytorch/pytorch/pull/978#issuecomment-333336017
